layouts/soarian/r4/clinical/diagnostics.html

```diff
@@ -0,0 +1,9 @@
+
+
+<p>The Diagnostic resources are concerned with the process of observing, testing, and diagnosing a patient. Note, once a diagnosis is confirmed, it would be recorded as a Condition so that it is part of the historical patient record.</p>
+
+<p>The Observation resource is primarily concerned with recording discrete values, answers, and results such as clinical laboratory data and vital signs while the DiagnosticReport resource typically provides a textual set of information and interpretation after performing a diagnostic service such as a Radiology report or Pathology report.</p>
```

Comment:
Break this statement into two sentences:
The Observation resource is primarily concerned with recording discrete values, answers, and results such as clinical laboratory data and vital signs.
The DiagnosticReport resource typically provides a textual set of information and interpretation after performing a diagnostic service such as a Radiology report or Pathology report.</p>
 
PRODOC-160857

Reply:
Made changes as suggested